website/docs/getting-started/setup/upgrade-to-business-edition/upgrade-docker-to-business-edition.mdx

```diff
@@ -2,10 +2,15 @@
 
 <HideElements/>
 
-Follow the steps below to upgrade Docker Community edition to Business edition:
+Follow the steps below to upgrade the Docker Community edition to the Business edition:
 
 <ul class="circle"> 
 
+<li> Create a backup of the CE data. Run the below command in the directory with <code>docker-compose.yml</code>. The output file is stored in <code>/appsmith-stacks/data/backup/</code>
+
+        docker-compose exec appsmith appsmithctl export_db
```

Comment:
```suggestion
        docker-compose exec appsmith appsmithctl backup
       /* Retrieve the salt and password located save it somewhere safe */
    docker-compose exec appsmith bash cat  /appsmith-stacks/configuration/docker.env | grep APPSMITH_ENCRYPTION_
```

Reply:
We'll backup the whole Appsmith instance (db+ docker.env + gitsync) not just the db.

Reply:
Got it. Do we have to add the additional steps of retrieving salt and password? I don't see it on the original docs here - https://docs.appsmith.com/getting-started/setup/upgrade-to-business-edition/docker#step-1-database-backup